gen/calc.py

```diff
@@ -1128,6 +1128,7 @@ def calculate_fault_domain_detect_contents(fault_domain_detect_filename):
         'mesos_log_retention_count': calculate_mesos_log_retention_count,
         'mesos_log_directory_max_files': calculate_mesos_log_directory_max_files,
         'dcos_version': DCOS_VERSION,
+        'dcos_distribution': 'open',
```

Comment:
Are we sure we just want to go with "open" here as the distribution? I would say we want to fully qualify the distribution we are talking about (i.e. "Mesosphere DC/OS"). Otherwise, I like JPs suggestion of calling this field `dcos_variant` instead of distribution. @jgehrcke @ToddGreenstein

Reply:
Let's keep distribution, and what values would you like to see @klueska to delineate EE vs Open?